flink-core/pom.xml

```diff
@@ -63,9 +63,20 @@ under the License.
 		<dependency>
 			<groupId>com.esotericsoftware.kryo</groupId>
 			<artifactId>kryo</artifactId>
+			<exclusions>
```

Comment:
I think if you add a dependency in the same pom (as below) you don't need the exclusion any more.